### ISSUE TITLE
Package contains string "start" in package name

### ISSUE THREAD

**Issue by imincik** [linz/linz-software-repository#39]
Package contains string "start" in package name. For example: `linz-bde-schema_1.12-start-linz~bionic_all.deb`

**Comment by strk** [comment]
See https://github.com/linz/linz-bde-schema/commit/596be55e9a9d6d6fcde146aa67b1e9dc0f73b15d - packages are named after `git describe`, but such packages should not be pushed to the `test` packagecloud repository, are them ?

**Comment by imincik** [comment]
Yes, we currently have https://packagecloud.io/linz/test/packages/ubuntu/bionic/linz-bde-schema_1.12-start-linz~bionic_all.deb in repo.

**Comment by strk** [comment]
I think we can delete that package manually as it should never build again. The problem was that the commit configuring github-actions so NOT to push packages with names ending in `start` was NOT included at the `1.12-start` tag point, so the action triggered by that tag did run anyway.

```
ommit 596be55e9a9d6d6fcde146aa67b1e9dc0f73b15d (HEAD -> master, origin/master, origin/HEAD)
Author: Sandro Santilli <strk@kbt.io>
Date:   Wed Jul 22 11:28:29 2020 +0200

    [ci] Do not build packages on pushing tags ending with "start"
    
    This is to allow defining tags on the first commit in master branch
    after a new stable branch is created. The tag would look like
    "1.12-start", so that `git describe` produces a version which is,
    according to dpkg --compare-version, *newer* than any version coming
    out from the most recent stable branch while still being *older*
    than the final .0 version being targetted in master branch (ie: 1.12.0)

commit a235d2cc77d5394cfc83087ab6db4de15713bb50
Author: Sandro Santilli <strk@kbt.io>
Date:   Tue Jul 14 10:48:25 2020 +0000

    [debian] Changelog update

commit 9b3f1c1ef6692d6de2bce015e172f7be582a1e7c (tag: 1.12-start)
Author: Sandro Santilli <strk@kbt.io>
Date:   Tue Jul 14 14:39:42 2020 +0200

    Target 1.12.0dev
```

**Comment by strk** [comment]
I'm closing this as non-actionable here. It's up to callers to deal with this.
